Stripe/STPRedirectContext.m

```diff
@@ -21,8 +22,15 @@
 typedef void (^STPBoolCompletionBlock)(BOOL success);
 
 @interface STPRedirectContext () <SFSafariViewControllerDelegate, STPURLCallbackListener>
-@property (nonatomic, copy) STPRedirectContextCompletionBlock completion;
-@property (nonatomic, strong) STPSource *source;
+/// Optional URL for a native app. This is passed directly to `UIApplication openURL:`, and if it fails this class falls back to `redirectUrl`
+@property (nonatomic, nullable, copy) NSURL *nativeRedirectUrl;
+/// The URL to redirect to, assuming `nativeRedirectUrl` is nil or fails to open. Cannot be nil if `nativeRedirectUrl` is.
+@property (nonatomic, nullable, copy) NSURL *redirectUrl;
```

Comment:
i would suggest calling it `webRedirectUrl` to reduce confusion with `nativeRedirectUrl`

Reply:
I'm not totally comfortable doing that. It's *almost* certainly a web url, but it doesn't actually have to be.